Project/Source Files/Services/Models/Recipe.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Recipt.swift
+//  NetguruCollegeApp
+//
+
+import Foundation
+
+struct Recipe {
+    let title: String
+    let imageURL: String
+    let ingredients: String
+    let recipeURL: String
+}
+
+extension Recipe: Codable {
+    enum CodingKeys: String, CodingKey {
+        case title = "title"
+        case imageURL = "thumbnail"
+        case ingredients = "ingredients"
+        case recipeURL = "href"
+    }
+}
+
+struct Recipes {
+    let title: String
+    let version: Float
```

Comment:
Which version and which title? It would be great to have documentation here too :)